### ISSUE TITLE
Player Name field reverts to Google Username

### ISSUE THREAD

**Issue by NotOnLand** [Mottokrosh/SheetIssues#16]
The 'Player Name' field always resets to my Google username after leaving and re-entering a character, no matter what device I change it on.

**Comment by Mottokrosh** [comment]
It may be that that field is auto-populated with the user data, for supposed convenience. :) I'll look into options.

**Comment by MythicParty** [comment]
Yup, same here.  Which ordinarily wouldn't be an issue, except the email account that I use for my gaming I always change the account name to my current character's name; don't worry this isn't me not being able to separate RPG from reality, it just helps with organization.
